priv/repo/migrations/20160401212441_insert_fallback_templates.exs

```diff
@@ -6,48 +6,18 @@ defmodule Cog.Repo.Migrations.InsertFallbackTemplates do
   def change do
     # That extra newline is there for a reason. Mustache spec strips newlines
     # following a standalone partial. No idea why.
-    Repo.insert!(%Template{
-      name: "json",
-      adapter: "slack",
-      source: """
-      ```
-      {{> json}}
 
-      ```
-      """
-    })
-
-    Repo.insert!(%Template{
-      name: "json",
-      adapter: "hipchat",
-      source: """
-      /code
-      {{> json}}
-      """
-    })
-
-    Repo.insert!(%Template{
-      name: "raw",
-      adapter: "any",
-      source: """
-      {{> json}}
-      """
-    })
-
-    Repo.insert!(%Template{
-      name: "json",
-      adapter: "any",
-      source: """
-      {{> json}}
-      """
-    })
-
-    Repo.insert!(%Template{
-      name: "text",
-      adapter: "any",
-      source: """
-      {{> text}}
-      """
-    })
+    execute """
+    INSERT INTO templates(id, name, adapter, source, inserted_at, updated_at)
+    VALUES
+    ('#{new_uuid}', 'json', 'slack',   '```\n{{> json}}\n\n```\n', now(), now()),
+    ('#{new_uuid}', 'json', 'hipchat', '/code\n{{> json}}\n\n',    now(), now()),
+    ('#{new_uuid}', 'raw',  'any',     '{{> json}}\n',           now(), now()),
+    ('#{new_uuid}', 'json', 'any',     '{{> json}}\n',           now(), now()),
+    ('#{new_uuid}', 'text', 'any',     '{{> text}}\n',           now(), now())
```

Comment:
I guess that could work... I was avoiding using any of the Ecto stuff since we got burned by using the models last time.